contracts/contracts/ZkEvmV2.sol

```diff
@@ -18,9 +18,13 @@ abstract contract ZkEvmV2 is Initializable, AccessControlUpgradeable, L1MessageS
   /// @dev DEPRECATED in favor of currentFinalizedState hash.
   uint256 public currentTimestamp;
 
+  /// @notice The most recent finalized L2 block number.
   uint256 public currentL2BlockNumber;
 
+  /// @notice The most recent L2 stateroot hash mapped by block number.
```

Comment:
Please remove the space here:  `stateroot` :)

Reply:
done